java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -822,6 +839,11 @@ class DatabaseImpl(
                 """.trimIndent(),
                 arrayOf(LARGEST_PRIMITIVE_TYPE_ID.toString()) // only entity collections.
             )
+            (storageKeys + updatedContainersStorageKeys).forEach {
+                storageKey -> notifyClients(StorageKeyParser.parse(storageKey)) {
```

Comment:
Nit: `storageKey ->` should go on the same line as the `forEach`

Reply:
done